api/components/icon.schema.json

```diff
@@ -23,7 +23,7 @@
         },
         "color": {
             "$ref": "defs/color.schema.json",
-            "default": "#000000"
+            "default": null
```

Comment:
Just to be sure, what a "Null" Icon color means ? Is it black if "null" is set ?

Reply:
If the icon color is null, its color will be defined according to the icon theme's value. This might get interesting when we start using the theme to style each component.

Reply:
Ok so maybe add a description to explain this and we're good !